app/views/rescue_codes/new.html.slim

```diff
@@ -0,0 +1,21 @@
+- title t('titles.rescue_codes')
+
+h1.heading = t('headings.rescue_codes')
+p
+  | Below are a set of rescue codes that we have generated for you.
+    You can use these codes to access your account in case you loose
+    access to your phone. Please download these codes and keep them
+    in a safe place.
```

Comment:
What do you think about removing the ability to download the codes? It would remove about half of the code in the RescueCodesController. I don't consider this a must-have. In fact, many sites I've seen specifically instruct the user to print or write them down on a piece of paper and keep it in a safe place. The point those sites make is that these codes should not be stored in a computer, and therefore should not be downloaded.

Also, it would be good to find out whether multiple codes are really necessary. Apple only gives you a single recovery key, meaning less writing down for the user.

Here is what Apple says about where to keep the key:
> Where should I keep my Recovery Key?

> Keep your Recovery Key in a secure place in your home, office, or other location. You should consider printing more than one copy, so that you can keep your key in more than one place. Your key will be easier to find if you ever need it, and you'll have a spare copy if one is ever lost or destroyed. You shouldn't store your Recovery Key on your device or computer, because that could give an unauthorized user instant access to your key.